### PR TITLE
SignIn

### PR DESCRIPTION
**Pull request by sheldondebra**
Login Form Completed

### CODE REVIEW

File: src/components/SignInForm.jsx [hunk 1]
```diff
@@ -0,0 +1,111 @@
...
+      setViewPassword((prevState) => !prevState);
+    };
+
+    return (
+      <div className="flex justify-between items-center bg-white rounded-md">
+        <Input
+          {...field}
+          type={viewPassword ? "text" : "password"}
+          placeholder="Enter your password"
+          className="border border-0 active:border-0 rounded-e-none bg-transparent"
```
Comment (yhawbee1) [inline src/components/SignInForm.jsx]: Remove one of the border classes

File: src/app/(auth)/sign-in/page.jsx [hunk 2]
```diff
@@ -1,9 +1,83 @@
...
 
 const SignUp = () => {
   return (
-    <div>Sign In</div>
-  )
-}
+    <>
+      <div className="flex">
+        <div className=" hidden md:flex h-screen w-[50%] bg-tbhgreen/80">
+          <iframe
```
Comment (yhawbee1) [inline src/app/(auth)/sign-in/page.jsx:13]: Fix the width of the video

File: src/components/SignInForm.jsx [hunk 3]
```diff
@@ -0,0 +1,111 @@
...
+
+import { zodResolver } from "@hookform/resolvers/zod";
+import { useForm } from "react-hook-form";
+import { z } from "zod";
+import { useState } from "react";
+import { Button } from "@/components/ui/button";
+import {
+  Form,
+  FormControl,
+  FormDescription,
```
Comment (yhawbee1) [inline src/components/SignInForm.jsx]: Remove the import if you won't use it

File: src/components/SignInForm.jsx [hunk 4]
```diff
@@ -0,0 +1,111 @@
...
+  FormMessage
+} from "@/components/ui/form";
+import { Input } from "@/components/ui/input";
+import { EyeIcon, EyeOffIcon } from "lucide-react"; // Import both icons
+
+const formSchema = z.object({
+  username: z.string().min(2, {
+    message: "Username must be at least 2 characters."
+  }),
+  password: z.string().min(6, {
```
Comment (yhawbee1) [inline src/components/SignInForm.jsx]: It should be 8 characters

File: src/components/SignInForm.jsx [hunk 5]
```diff
@@ -0,0 +1,112 @@
...
+                  type="text"
+                  placeholder="Enter your username"
+                  className="dark:text-gray-100 dark:bg-transparent"
+                />
+              </FormControl>
+              <FormMessage />
+            </FormItem>
+          )}
+        />
+        <FormField
```
Comment (yhawbee1) [inline src/components/SignInForm.jsx:106]: The form requires only one <FormField />

File: src/components/SignInForm.jsx [hunk 6]
```diff
@@ -0,0 +1,112 @@
...
+          name="username"
+          render={({ field }) => (
+            <FormItem>
+              <FormLabel>Username</FormLabel>
+              <FormControl>
+                <Input
+                  {...field}
+                  type="text"
+                  placeholder="Enter your username"
+                  className="dark:text-gray-100 dark:bg-transparent"
```
Comment (yhawbee1) [inline src/components/SignInForm.jsx]: The border color of the two fields are not the same

File: src/components/SignInForm.jsx (hunk 1 above)
Comment (sheldondebra) [reply src/components/SignInForm.jsx]: Fixed Sir

File: src/app/(auth)/sign-in/page.jsx (hunk 2 above)
Comment (sheldondebra) [reply src/app/(auth)/sign-in/page.jsx:13]: I will work on the video during the next sprint

File: src/components/SignInForm.jsx (hunk 3 above)
Comment (sheldondebra) [reply src/components/SignInForm.jsx]: Fixed

File: src/components/SignInForm.jsx (hunk 4 above)
Comment (sheldondebra) [reply src/components/SignInForm.jsx]: Fixed

File: src/components/SignInForm.jsx (hunk 6 above)
Comment (sheldondebra) [reply src/components/SignInForm.jsx]: Fixed

**Pull request merged**